modules/configuring-firewall.adoc

```diff
@@ -5,16 +5,93 @@
 [id="configuring-firewall_{context}"]
 = Configuring your firewall for {product-title}
 
-Before you install {product-title}, you must configure your firewall to access
-Red Hat Insights.
+Before you install {product-title}, you must configure your firewall to grant access to the sites that {product-title} requires.
 
 .Procedure
 
-* Set your firewall to allow the following host names and ports on the outgoing
-network firewall:
+. Whitelist the following registry URLS:
 +
-----
-cert-api.access.redhat.com:443
-api.access.redhat.com:443
-infogw.api.openshift.com:443
-----
+[cols="3,4",options="header"]
+|===
+|URL | Function
+|`registry.redhat.io`
+|Provides core container images
+
+|`*.quay.io`
+|Provides core container images
+
+|`sso.redhat.com`
+|The `https://cloud.redhat.com/openshift` site uses authentication from `sso.redhat.com`
+|===
+
+. Whitelist any site that provides resources for a language or framework that your builds require.
+
+. If you do not disable Telemetry, you must grant access to the following host names and ports to access Red Hat Insights:
++
+[cols="3,4",options="header"]
+|===
+|URL | Function
+
+|`cert-api.access.redhat.com`
+|Required for telemetry
+
+|`api.access.redhat.com`
+|Required for telemetry
+
+|`infogw.api.openshift.com`
+|Required for telemetry
+|===
+
+. If you use Amazon Web Services (AWS), Microsoft Azure, or Google Cloud Platform (GCP) to host your cluster, you must grant access URLs that provide the cloud provider API and DNS for that cloud:
++
+[cols="2a,8a,8a",options="header"]
+|===
+|Cloud |URL |Function
+
+|AWS
+|`*.amazonaws.com`
+|Required to access AWS services and resources. Review the link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS Service Endpoints] in the AWS documentation to determine the exact endpoints to allow for the regions that you use.
+
+.2+|GCP
+|`*.googleapis.com`
+|Required to access GCP services and resources. Review link:https://cloud.google.com/endpoints/[Cloud Endpoints] in the GCP documentation to determine the endpoints to allow for your APIs.
+
+|`accounts.google.com`
+| Required to access your GCP account
+
+|Azure
+|`management.azure.com`
+|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API Reference] in the Azure documentation to determine the endpoints to allow for your APIs.
+
+|===
+
+. Whitelist the following URLs:
++
+[cols="3,4",options="header"]
+|===
+|URL | Function
+
+|`mirror.openshift.com`
+|Required to access mirrored installation content and images
+
+|`*.cloudfront.net`
+|Required by the Quay CDN to provide edge delivery of Quay.io images
+
+|`*.apps`
+|Required to access the default cluster routes unless you set an ingress wildcard during installation.
```

Comment:
This has a period but the others in the table don't. Be consistent either with or without. And maybe look at being consistent across tables in this module.